source/User_Guide/Marketing_Campaigns/create-and-manage-unsubscribe-groups.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+weight: 90
+title: Create and Manage Unsubscribe Groups
+seo:
+  title: Create and Manage Unsubscribe Groups
+  description: Suppression Manager helps you to define and manage unsubscribe groups to keep you out of the spam folder.
+  keywords: unsubscribe groups, suppressions, manage unsubscribe groups, delete unsubscribe groups
+navigation:
+  show: true
+---
+
+* [Create an Unsubscribe Group](#-Create-an-Unsubscribe-Group)
+* [Add recipients to an Unsubscribe Group](#-Add-recipients-to-an-Unsubscribe-Group)
+* [Managing Unsubscribe Groups](#-Managing-Unsubscribe-Groups)
+   * [Exporting an Unsubscribe Group List](#-Exporting-an-Unsubscribe-Group-List)
+   * [Editing an Unsubscribe Group](#-Editing-an-Unsubscribe-Group)
+   * [Deleting an Unsubscribe Group](#-Deleting-an-Unsubscribe-Group)
+
+Adding Unsubscribe Groups to your emails makes it easy to honor your recipients' email preferences and protects your sender reputation by complying with anti-spam legislation. 
+ 
+{% anchor h2 %}
+Create an Unsubscribe Group
+{% endanchor %}
+
+*To create an Unsubscribe Group:*
+
+1. Select **Marketing Campaigns** and then click **Unsubscribe Groups**.
+1. Click **Add Unsubscribe Group**. 
+1. Add a *Group Name* and *Group Description*. 
+
+{% info %}
+Make sure your Group Names and Group Descriptions are customer-friendly. This is what your recipients will see. 
+{% endinfo %}
+
+1. Select the checkbox if you want the Unsubscribe Group to display on the unsubscribe preferences page. 
+1. click **Save Unsubscribe Group**. 
+
+{% info %}
+To view the unsubscribe preferences page, select the action menu next to an Unsubscribe Group and then click **Preview**. Toggle to the Unsubscribe Preferences tab to view all of the options listed. {% endinfo %}
```

Comment:
This update will now move the preview to be side by side as they are editing the fields, so this is no longer required for them to preview them. That said, once a group is created, they will be viewed as cards on the main Unsubscribe Groups index page, and there's now a 3 dot menu on each card, with a preview option as well.